network-api/networkapi/wagtailpages/utils.py

```diff
@@ -1,10 +1,18 @@
+import ntpath
```

Comment:
Is there an issue to take this out of the other code path(s)?

Reply:
We use this to extract the base filename from an image source if an explicit name is not provided: `image_name = ntpath.basename(img_src)`. This just makes sure the Wagtail Image has a "title" that matches the filename. The title can be anything but if we don't know the context of the image being created, it's probably safest to fallback to the filename